code/datums/mutations/body.dm

```diff
@@ -193,7 +193,7 @@
 	desc = "A strange genome, believing to be what differentiates monkeys from humans."
 	text_gain_indication = "You feel unusually monkey-like."
 	text_lose_indication = "You feel like your old self."
-	quality = NEGATIVE
+	quality = POSITIVE
```

Comment:
Idk if I like this fix, perhaps instead we just add a var or something to mutations that says `do_not_remove_on_aheal`?

Reply:
if we don't have a bitflag variable (like `flags_1`, `obj_flags`) for mutations it would be nice if we could create one for stuff like this rather than have a generic boolean variable

Reply:
> if we don't have a bitflag variable (like `flags_1`, `obj_flags`) for mutations it would be nice if we could create one for stuff like this rather than have a boolean variable take up space

The only bitflag after looking seems to be quality.

Reply:
and datum flags

Reply:
![image](https://github.com/tgstation/tgstation/assets/34697715/96416ae6-9622-4ad6-86e1-2629c09911ab)

these three (and the potential fourth) should be rolled into one variable but you don't have to do that now, just noting it down for future reference/refactor material

Reply:
> ![image](https://private-user-images.githubusercontent.com/34697715/318993516-96416ae6-9622-4ad6-86e1-2629c09911ab.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIxMDQwOTgsIm5iZiI6MTcxMjEwMzc5OCwicGF0aCI6Ii8zNDY5NzcxNS8zMTg5OTM1MTYtOTY0MTZhZTYtOTYyMi00YWQ2LTg2ZTEtMjYyOWMwOTkxMWFiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAzVDAwMjMxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmMzFjMDI3ZGNlOGE5NTc1OGUzYjkwNDMwOWY2ZTAyNDU0ZjdiYTM0MTRlNmQ0ODMwNTBkNTg0OTMxOWU4NzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3-qBqkJZLfZ0k-W-QEShMuSJQNJE447vuPVJzednAi0)
> 
> these three (and the potential fourth) should be rolled into one variable but you don't have to do that now, just noting it down for future reference/refactor material

and probably var/locked too.